how-to-docs/namespace-how-to.md

```diff
@@ -1,23 +1,13 @@
-## How do I create a namespace in the UK cluster?
-Please visit the [ACP-Kube](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube) repo on Gitlab to create a namespace.
+# How to get a new Namespace in a UK cluster
 
-+ Clusters are defined by the folders within the `policies/` directory, with each cluster having a subdirectory labelled `namespaces`.
+1. Log into the [Platform Hub][platform hub link].
```

Comment:
Are we able to add the link now?

Reply:
Link is at the bottom (in reference style)